Kconfig.zephyr

```diff
@@ -111,6 +111,17 @@ config FLASH_LOAD_OFFSET
 
 	  If unsure, leave at the default value 0.
 
+config FLASH_LOAD_OVERHEAD_SIZE
```

Comment:
The purpose of this seems similar to that of `ROM_START_OFFSET`. Would it be appropriate to call it something like `ROM_END_OFFSET`?

Reply:
Fixed